sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/DotCoverReportParser.java

```diff
@@ -36,7 +41,7 @@
   private static final Pattern COVERED_LINES_PATTERN_1 = Pattern.compile(
     ".*<script type=\"text/javascript\">\\s*+highlightRanges\\(\\[(.*?)\\]\\);\\s*+</script>.*",
     Pattern.DOTALL);
-  private static final Pattern COVERED_LINES_PATTERN_2 = Pattern.compile("\\[(\\d++),\\d++,\\d++,\\d++,(\\d++)\\]");
+  private static final Pattern COVERED_LINES_PATTERN_2 = Pattern.compile("\\[(\\d++),\\d++,(\\d++),\\d++,(\\d++)\\]");
```

Comment:
why was this change needed?

Reply:
In order to be able to extract the endLine with the matcher